quipucords/fingerprinter/jboss_web_server.py

```diff
@@ -1,6 +1,6 @@
 """Ingests raw facts to determine the status of JBoss Web Server on system."""
 
-from api.models import Product
+from api.deployments_report.model import Product
```

Comment:
outside of api package we should be importing all models from api.models. Please change it back.